draft-ietf-httpbis-digest-headers.md

```diff
@@ -358,152 +438,213 @@ Examples:
+    * Digest Dictionary key: unixcksum
+    * Digest List item: unixcksum
     * Reference: [UNIX], this document.
     * Status: insecure
 
-  adler32
-  : * Digest Algorithm: adler32
+  ADLER
+  : * Algorithm: ADLER32
     * Description: The ADLER32 algorithm [RFC1950].
-      The 32-bit output is encoded in hexadecimal (using
-      between 1 and 8 ASCII characters from 0-9, A-F, and a-f; leading 0's are
-      allowed). For example, adler32=03da0195 and adler32=3DA0195 are both valid
-      checksums for the 4-byte message "Wiki". This algorithm is obsoleted and
-      SHOULD NOT be used.
+    * Digest-algorithm token: adler32
+    * Digest-algorithm output format: The 32-bit output is encoded in
+      hexadecimal (using between 1 and 8 ASCII characters from 0-9, A-F,
+      and a-f; leading 0's are allowed).
+    * Digest Dictionary key: adler32
+    * Digest List item: adler32
     * Reference: [RFC1950], this document.
     * Status: insecure
 
-  crc32c
-  : * Digest Algorithm: crc32c
+  CRC32C
+  : * Algorithm: CRC32C
     * Description: The CRC32c algorithm {{!RFC4960}}.
-      The 32-bit output is encoded in hexadecimal (using between 1 and 8
-      ASCII characters from 0-9, A-F, and a-f; leading 0's are allowed). For
-      example, crc32c=0a72a4df and crc32c=A72A4DF are both valid checksums for the
-      3-byte message "dog".
+    * Digest-algorithm token: crc32c
+    * Digest-algorithm output format: The 32-bit output is encoded in
+      hexadecimal (using between 1 and 8 ASCII characters from 0-9, A-F,
+      and a-f; leading 0's are allowed).
+    * Digest Dictionary key: crc32c
+    * Digest List item: crc32c
     * Reference: {{!RFC4960}} appendix B, this document.
     * Status: insecure
 
-# Using Digest in State-Changing Requests {#state-changing-requests}
+  CONTENTMD5
+  : * Algorithm: contentMD5
+    * Description: {{Section 5 of RFC3230}} defined the "contentMD5" token to be used only in Want-Digest.
+  This token is obsoleted by this document.
+    * Digest-algorithm token: Not applicable
+    * Digest-algorithm output format: Not applicable
+    * Digest Dictionary key: Not applicable
+    * Digest List item: Not applicable
+    * Reference: {{Section 5 of RFC3230}}, this document.
+    * Status: obsolete
+
+# Using Representation Digests in State-Changing Requests {#state-changing-requests}
 
 When the representation enclosed in a state-changing request
 does not describe the target resource,
 the representation digest MUST be computed on the
 representation-data.
 This is the only possible choice because representation digest requires complete
-representation metadata (see {{digest}}).
+representation metadata (see {{representation-digest}}).
 
 In responses,
 
 - if the representation describes the status of the request,
-  `Digest` MUST be computed on the enclosed representation
+  `Digest` or `Representation-Digest` MUST be computed on the enclosed representation
```

Comment:
wdyt of referencing representation-data digest which applies to both fields?

Reply:
I like it!